dom0-updates/qubes-dom0-update

```diff
@@ -201,9 +206,17 @@ qvm-run --nogui -q $UPDATEVM 'rm -rf /var/lib/qubes/dom0-updates/etc' || exit 1
 tar c /var/lib/rpm /etc/yum.repos.d /etc/yum.conf /etc/dnf/dnf.conf 2>/dev/null | \
    qvm-run --nogui -q --pass-io "$UPDATEVM" 'LC_MESSAGES=C tar x -C /var/lib/qubes/dom0-updates 2>&1 | grep -v -E "s in the future"'
 
-qvm-run $QVMRUN_OPTS --pass-io $UPDATEVM "script --quiet --return --command '/usr/lib/qubes/qubes-download-dom0-updates.sh --doit --nogui $(escape_args "${ALL_OPTS[@]}")' /dev/null" < /dev/null
+CMD="/usr/lib/qubes/qubes-download-dom0-updates.sh --doit --nogui $(escape_args "${ALL_OPTS[@]}")"
+# Use 'script' to fake a terminal, so that we are shown sync and download
+# progress indicators. However, do it only if we are running in terminal
+# ourselves.
+if [ -t 1 ] && [ -t 2 ]; then
```

Comment:
Are you sure about checking stderr? This will make `qubes-dom0-update 2>/dev/null` behave interestingly ;)

Reply:
From what I understand, `qubes-dom0-update 2>/dev/null` will silence the errors, and also disable showing the progress bar. Am I right?

`script` will intercept both stdout and stderr, so if I just check stdout here, `qubes-dom0-update 2>/dev/null` will have no effect, which is also surprising.

My thinking was that as soon as someone starts doing something "custom" with the outputs, we should err on the side of caution, and disable our terminal hack.

Reply:
Ok, makes sense.